tests/modules/teams/resources/test_options.py

```diff
@@ -0,0 +1,34 @@
+# pylint: disable=missing-docstring
+import pytest
+
+@pytest.mark.parametrize('path,status_code', (
+    ('/api/v1/teams/', 401),
+    ('/api/v1/users/1', 401),
+))
+def test_users_options_unauthorized(path, status_code, flask_app_client):
```

Comment:
I think it is worth to rename these functions. `users` in not an appropriate name in `teams` module.